include/MFSegments.h

```diff
@@ -33,7 +33,7 @@ class MFSegments
 
 private:
     LedControl _ledControl;
-    bool _initialized;
+    //bool _initialized;
```

Comment:
Please remove the line instead of commenting it

Reply:
Oopsie.. forgot that.